scalafmt-cli/shared/src/main/scala/org/scalafmt/cli/CliOptions.scala

```diff
@@ -40,12 +40,14 @@ object CliOptions {
       if (parsed.quiet) Output.NoopStream
       else {
         val usesOut = parsed.stdIn || parsed.writeMode.usesOut
-        val cons = if (usesOut) System.console() else null
-        if (cons ne null) new Output.FromWriter(cons.writer())
-        else new Output.FromStream(
-          if (parsed.noStdErr || !usesOut) parsed.common.out
-          else parsed.common.err,
-        )
+        val writerMaybe = if (usesOut) getConsoleWriter() else None
```

Comment:
please change to `consWriter` or `consWriterOpt`, or, if you really prefer, `maybeCons[ole]Writer`. other than a few standard geeky suffixes like Opt a variable or class name should be a correct English noun phrase (and methods a verb phrase).